scheduler/cups-driverd.cxx

```diff
@@ -2914,3 +2927,74 @@ regex_string(const char *s)		/* I - String to compare */
 
   return (NULL);
 }
+
+void *
```

Comment:
@pranjanpr IIUC you only return NULL and I'm not sure why it is needed. You only check ```retval``` later, but never the return value of ```cupsFileGetsCall()```).
IMO you can drop ```void *``` and ```return NULL;``` and use only ```void``` instead.